config/labels/label_descriptions.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I think the `label` and `checker_labels` keywords are a little bit redundant for me because we are in the `labels` directory. What do you think on the following structure:
```sh
|- labels
  |- analyzers
    |- clangsa.json
    |- clang-tidy.json
  |- descriptions.json
```